.eslintrc.json

```diff
@@ -0,0 +1,43 @@
+{
+    "env": {
+        "es6": true,
+        "node": true,
+        "jasmine": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "no-unused-vars": "off",
```

Comment:
Sanity: should this be on? Or the typescript version below take precedence?

Reply:
This item is actually related to line 24. It is a way for the typescript not to complain about imports that are used only for typing.

Example:

```typescript
import { AnyType } from 'model'

Promise<AnyType[]>
```